addons/binding/org.openhab.binding.jeelink/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+                <description>@text/parameter.updateinterval.description</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="sensorTimeout" type="integer" required="true" min="5" max="3600" unit="s" step="5">
+                <label>@text/parameter.sensortimeout.label</label>
+                <description>@text/parameter.sensortimeout.description</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="bufferSize" type="integer" required="true" min="0" max="100">
+                <label>@text/parameter.buffersize.label</label>
+                <description>@text/parameter.buffersize.description</description>
+				<default>20</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Current Watt Channel Type -->
+	<channel-type id="current-watt-channel">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.current-watt-channel.label</label>
+		<description>@text/channel-type.current-watt-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Max Watt Channel Type -->
+	<channel-type id="max-watt-channel">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.max-watt-channel.label</label>
+		<description>@text/channel-type.max-watt-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Total Consumption Channel Type -->
+	<channel-type id="consumption-total-channel">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.consumption-total-channel.label</label>
+		<description>@text/channel-type.consumption-total-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Appliance On Time Channel Type -->
+	<channel-type id="appliance-time-channel">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.appliance-time-channel.label</label>
+		<description>@text/channel-type.appliance-time-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Sensor On Time Channel Type -->
+	<channel-type id="sensor-time-channel">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.sensor-time-channel.label</label>
+		<description>@text/channel-type.sensor-time-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Resets Channel Type -->
+	<channel-type id="resets-channel">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.resets-channel.label</label>
+		<description>@text/channel-type.resets-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Sensor ID Channel Type -->
+	<channel-type id="sensor-id-channel">
+		<item-type>String</item-type>
+		<label>@text/channel-type.sensor-id-channel.label</label>
+		<description>@text/channel-type.sensor-id-channel.description</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<!-- Sensor Type Channel Type -->
+	<channel-type id="sensor-type-channel">
```

Comment:
This is part of the protocol and is reported by the sensor with either 1 or 2 as a value. I have no idea what it means but thought it would be better to make it available for people that did. I will remove it.